### PR TITLE
docs(config) Updated config - Added usage in main.ts

### PR DESCRIPTION
**Pull request by AntonyGarand**
Added usage of the config service in the main.ts file

## PR Checklist
Please check if your PR fulfills the following requirements:

[X] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[X] Other... Please describe: Documentation
```

## What is the current behavior?

Issue Number: https://github.com/nestjs/nest/issues/530

## Does this PR introduce a breaking change?
```
[ ] Yes
[X] No
```

## Other information

### CODE REVIEW

File: content/techniques/configuration.md [hunk 1]
```diff
@@ -115,6 +115,20 @@ export class AppService {
 
 > info **Hint** Instead of importing `ConfigModule` in each module, you can alternatively declare `ConfigModule` as a [global module](https://docs.nestjs.com/modules#global-modules).
 
+#### Using the Config Service in `main.ts`
+While our config is a stored in a service, it can still be used in the main.ts file.  
+This way, you can use it to store variables such as the application port or the CORS host.
+
+To access it, you must use `app.get`, followed by the name of your service:
+```typescript
+const config = app.get("ConfigService");
```
Comment (kamilmysliwiec) [inline content/techniques/configuration.md]: Why didn't you use `ConfigService`? (instead of the `"ConfigService"` string)

Comment (AntonyGarand) [reply content/techniques/configuration.md]: I didn't know we could do this actually

I'll update the PR with the config service directly

Comment (kamilmysliwiec) [reply content/techniques/configuration.md]: Ping

Comment (AntonyGarand) [reply content/techniques/configuration.md]: Updated! Sorry for the delay

**Pull request merged**